### ISSUE TITLE
Long PAPI event name will crash IPM

### ISSUE THREAD

**Issue by jscarretero** [nerscadmin/IPM#19]
Dear Admin,
I think I may have found a problem when telling IPM to profile long PAPI event names.

For example, profiling a valid native PAPI event through IPM_HPM like
export IPM_HPM=DTLB_LOAD_MISSES:MISS_CAUSES_A_WALK 
will make IPM crash.

```
*** buffer overflow detected ***: bin/mpi terminated
======= Backtrace: =========
/lib/x86_64-linux-gnu/libc.so.6(+0x77725)[0x7f5317326725]
/lib/x86_64-linux-gnu/libc.so.6(__fortify_fail+0x5c)[0x7f53173c789c]
/lib/x86_64-linux-gnu/libc.so.6(+0x1168a0)[0x7f53173c58a0]
/lib/x86_64-linux-gnu/libc.so.6(+0x115bf2)[0x7f53173c4bf2]
/home/jcarrete/installations/IPM/lib/libipm.so(ipm_check_env+0x2bc)[0x7f5317b7579c]
/home/jcarrete/installations/IPM/lib/libipm.so(ipm_get_env+0x231)[0x7f5317b75c31]
/home/jcarrete/installations/IPM/lib/libipm.so(ipm_init+0xac)[0x7f5317b74e3c]
/home/jcarrete/installations/IPM/lib/libipm.so(MPI_Init+0x23)[0x7f5317b7c673]
bin/mpi[0x400c56]
/lib/x86_64-linux-gnu/libc.so.6(__libc_start_main+0xf0)[0x7f53172cf830]
bin/mpi[0x400fb9]
======= Memory map: ========
```

I have modified the size for `MAXSIZE_PAPI_EVTNAME` in `/include/ipm_sizes.h` to acommodate for bigger event names, and it seems to work, but I don't know the implications of this change.

Javi Carretero

**Comment by njwright** [comment]
Hi

Thanks for letting us know. I don't think that the implications are large,
just a little more memory used.

Nick.

**Comment by cdaley** [comment]
Fixed in commit f9ec0a4. We increased the number of characters per label from 32 to 128.
